front/src/applications/stdcm/views/OSRDSTDCM.tsx

```diff
@@ -10,10 +10,9 @@ import StdcmRequestModal from './StdcmRequestModal';
 export default function OSRDSTDCM() {
   const dispatch = useDispatch();
   const mode = useSelector(getMode);
-  const [currentStdcmRequestStatus, setCurrentStdcmRequestStatus] = useState(
-    STDCM_REQUEST_STATUS.idle
-  );
-  const [, setCurrentStdcmRequestResults] = useState(null);
+  const [currentStdcmRequestStatus, setCurrentStdcmRequestStatus] = useState<
+    'IDLE' | 'PENDING' | 'SUCCESS' | 'REJECTED' | 'CANCELED' | 'NORESULTS'
+  >(STDCM_REQUEST_STATUS.idle);
```

Comment:
With the type suggested above, you could do
```suggestion
  const [currentStdcmRequestStatus, setCurrentStdcmRequestStatus] = useState<StdcmRequestStatusValues>(STDCM_REQUEST_STATUS.idle);
```

Reply:
done